submissions/description/broken_authentication_and_session_management/failure_to_invalidate_session/on_password_change/guidance.md

```diff
@@ -0,0 +1,7 @@
+# Guidance
+
+Provide a 1-2 sentence description - see [CVE Project's Key Details Phrasing Guide](http://cveproject.github.io/docs/content/key-details-phrasing.pdf) for tips. Provide a step-by-step walkthrough on how to access the vulnerable authentication mechanism and how to exploit the vulnerability to obtain access or perform a function that is intended to enforce authentication. Adding a dot-pointed walkthrough with relevant screenshots will speed triage time and result in faster rewards!
```

Comment:
I was kinda surprised at how much of this text would actually fit as a hint next to the description field, but it almost does, depending on where the hint text begins. It's pretty big though - can we reduce this at all?

Reply:
Yeah, the default guidance text is nice and short. I'll be more mindful of the length of it when I add in specific info.
I'll have a go now at reducing the size of this one :)